src/Components/Contacts/index.js

```diff
@@ -0,0 +1,14 @@
+import React, { memo } from "react";
+import ContactItem from "./ContactItem";
+
+const Contacts = ({contacts}) => {
+    return (
+        contacts.map((contact) => (
+            <div key={contact.id}>
+                <ContactItem {...contact}/>
```

Comment:
If your component is closing itself and it contains just one line the closing "/" need to have one space before it, like

<ContactItem {...contact} />